dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumEventExt.kt

```diff
@@ -55,14 +55,14 @@ internal fun RumResourceKind.toSchemaType(): ResourceEvent.ResourceType {
     }
 }
 
-internal fun RumErrorSource.toSchemaSource(): ErrorEvent.Source {
+internal fun RumErrorSource.toSchemaSource(): ErrorEvent.ErrorSource {
```

Comment:
where this change (`ErrorEvent.Source` -> `ErrorEvent.ErrorSource` renaming) comes from? In the schema this property still called `source`.

Reply:
That is a very good question. I had assumed either the schema changed for that or that the generator was attempting to avoid a name collision.

Reply:
Yup that's it